network-api/networkapi/wagtailpages/forms.py

```diff
@@ -39,3 +39,15 @@ def clean(self):
             self.add_error('topics', 'Please select 2 topics max.')
 
         return cleaned_data
+
+
+# Similar validation to the BlogPageForm, but for the Blog Index page
+# and its field 'related_topics'.
+class BlogIndexTopicCountValidationForm(WagtailAdminPageForm):
+    def clean(self):
+        cleaned_data = super(BlogIndexTopicCountValidationForm, self).clean()
```

Comment:
**Suggestion** I think you don't need to repeat the class name in the `super()` call since Python 3.6. `super().clean()` should be enough.

Reply:
This does work! So I updated both of the custom form classes